qiskit/user_config.py

```diff
@@ -29,14 +30,17 @@ class UserConfig:
     [default]
     circuit_drawer = mpl
     circuit_mpl_style = default
+    circuit_mpl_style_path = /home/user/styles;/home/user
+    transpile_optimization_level = 2
+    suppress_packaging_warnings = True
```

Comment:
These seems unrelated. 
```suggestion
```

Reply:
These are all config file options. I just added them to the doc string.

Reply:
You can add them as comments too, with their default value.